Gemfile.lock

```diff
@@ -117,20 +116,14 @@ GEM
     mini_mime (1.1.2)
     mini_portile2 (2.8.0)
     minitest (5.16.2)
-    net-imap (0.2.3)
-      digest
+    net-imap (0.3.1)
```

Comment:
how did this get updated bro?

Reply:
just commit the changes that are made , we need to push all the changes which are not relevant i assume its because of the bundler version changes

Reply:
This got updated when I ran bundle install. Looks like nested dependency changes to me. Should I revert this back? @daksh1103 @rishirajkumar97

Reply:
this might be changed due to nested dependency changes

Reply:
or since we had some configurations to auto update vulnerabilities  in the repo @ashwin47  is it due to that?